packages/apple-pay-integration/src/apple-pay-button-strategy.ts

```diff
@@ -17,8 +18,9 @@ import {
     ShippingOption,
     StoreConfig,
 } from '@bigcommerce/checkout-sdk/payment-integration-api';
-
-import { WithApplePayButtonInitializeOptions } from './apple-pay-button-initialize-options';
+import ApplePayButtonInitializeOptions, {
```

Comment:
Extra space should be added before this line due to linter warning

Reply:
fixed